scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -116,6 +123,8 @@ class PublishVisibility(Enum):
 EXCLUDE_PREFIX = "!"
 FULL_ASSET_NAME_DELIMITER = "/"
 FULL_ASSET_NAME_TEMPLATE = "{type}/{name}/{version}"
+GENERIC_ASSET_TYPES = [AssetType.PROMPT]
+NON_GENERIC_ASSET_TYPES = [AssetType.COMPONENT, AssetType.DATA, AssetType.ENVIRONMENT, AssetType.MODEL]

```

Comment:
Please consider something like `STANDARD_ASSET_TYPES` instead of `NON_GENERIC_ASSET_TYPES`.

Reply:
Updated, thanks